driver/remote/factory.go

```diff
@@ -87,6 +88,12 @@ func (f *factory) New(ctx context.Context, cfg driver.InitConfig) (driver.Driver
 				return nil, err
 			}
 			d.defaultLoad = parsed
+		case "timeout":
```

Comment:
Why is this specific for remote driver?

Reply:
In the `driver/kubernetes/factory.go` New function the option parsing is done in `processDriverOpts`, the `driver/docker-container/factory.go` New function missing this option parsing seems like an oversight possibly, I added the timeout option parsing there too, the `driver/docker/factory.go` New function doesn't seem to do any option parsing at all so not doing any timeout parsing there is consistent with the existing code but I don't know if it is okay.